frontend/public/components/dashboard/dashboards-page/cluster-dashboard/status-card.tsx

```diff
@@ -139,8 +145,22 @@ const ClusterAlerts = withDashboardResources(
     }
 
     if (alertsLoaded && !_.isEmpty(alerts)) {
+      const filteredAlerts = isClusterDashboard
+        ? alerts
+        : alerts.filter((alert) => {
+            let containsLabel = false;
+            _.each(labelFilters, (filterValue, filterKey) => {
+              if (alert?.labels?.[filterKey] === filterValue) {
+                containsLabel = true;
+                return;
+              }
+            });
+            return containsLabel;
```

Comment:
```suggestion
            return _.every(labelFilters, (filterValue, filterKey) => {
              return alert?.labels?.[filterKey] === filterValue);
            });
```

Reply:
@spadgett thanks, fixed the filtering logic.